test_data/std/fp4.asm

```diff
@@ -0,0 +1,8 @@
+use std::math::fp4::test;
+
+machine Main {
+    test::add();
+    test::sub();
+    test::mul();
+    test::inverse()[0];
+}
```

Comment:
Instead of this, we should have the analog of [this](https://github.com/powdr-labs/powdr/blob/3d4fedfcc47f30a85b6b4ef8f9dfeeaf004c1e3e/pipeline/tests/powdr_std.rs#L318), no? Then we can test that it works on all 3 fields as well.

Reply:
Also, fixed the documentation